README.md

```diff
@@ -1,259 +1,16 @@
 # ForgAE
 
+[![npm version](https://badge.fury.io/js/forgae.svg)](https://badge.fury.io/js/forgae)
+
 **ForgAE** is an aeternity framework which helps with setting up a project.
 The framework makes the development of smart contracts in the aeternity network pretty easy. It provides commands for compilation, deployment of smart contracts, running a local node, local compiler and unit testing the contracts.
```

Comment:
probably here we should explain the splitting of the packages, something like "forgae has 5 packages,  forgae-lib will give you access yo deployer " and so on. Some brief information about the packages